crates/polars-core/src/series/ops/reshape.rs

```diff
@@ -56,6 +57,25 @@ impl Series {
         }
     }
 
+    /// TODO: Move this somewhere else?
+    pub fn list_offsets_and_validities_recursive(
```

Comment:
This always outputs Vecs with 1 element.

Reply:
Not necessarily - it returns Vecs with a number of elements corresponding to the nesting level - so e.g. for a `List(List(Float64))` - 

```python
print((pl.Series([[[1]]]) / pl.Series([1])).dtype)
List(List(Float64))

[crates/polars-core/src/series/arithmetic/list_borrowed.rs:93:17] lhs.list_offsets_and_validities_recursive().0 = [
    OffsetsBuffer(
        [
            0,
            1,
        ],
    ),
    OffsetsBuffer(
        [
            0,
            1,
        ],
    ),
]
```